s/babylon/camera/spectator-camera.ts

```diff
@@ -32,74 +36,48 @@ export function makeSpectatorCamera({
 	const transformB = new TransformNode("camB", scene)
 	const camera = (() => {
 		const name = "spectator camera"
-		const position = new Vector3(0, 5, 0)
+		const position = new Vector3(...startPosition)
```

Comment:
i believe this is the wrong place to apply the startPosition

we actually want to set this on `transformA`